service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -76,9 +77,17 @@
     private String publicKey;
     @Value("${liqpay.private.key}")
     private String privateKey;
+    @Value("${ubs.viber.bot.uri}")
+    private String viberBotUri;
+    @Value("${ubs.bot.name}")
+    private String telegramBotName;
```

Comment:
Better create  ConfigProperties class instead of injecting @ Value. Example of creation #621

Reply:
@Zingerd Our architecture of app does not provide access to the Config Properties from Service layer.

Reply:
@Zingered Architecture of application does not provide access for Configuration properties from Service layer.